binding/HarfBuzzSharp.Shared/HarfBuzzApi.cs

```diff
@@ -118,6 +166,15 @@ internal class HarfBuzzApi
 		public extern static hb_bool_t hb_font_get_glyph_h_origin (hb_font_t font, hb_codepoint_t glyph, out hb_position_t x, out hb_position_t y);
 		[DllImport (HARFBUZZ, CallingConvention = CallingConvention.Cdecl)]
 		public extern static hb_bool_t hb_font_get_glyph_v_origin (hb_font_t font, hb_codepoint_t glyph, out hb_position_t x, out hb_position_t y);
+		[DllImport (HARFBUZZ, CallingConvention = CallingConvention.Cdecl)]
+		public extern static void hb_font_set_funcs (IntPtr font, hb_font_funcs_t klass, IntPtr font_data, ReleaseDelegateProxyDelegate destroy);
+		[DllImport (HARFBUZZ, CallingConvention = CallingConvention.Cdecl)]
+		public extern static hb_font_t hb_font_get_parent (hb_font_t font);
+		[DllImport (HARFBUZZ, CallingConvention = CallingConvention.Cdecl)]
+		public extern static hb_bool_t hb_font_get_glyph_name (hb_font_t font, hb_codepoint_t glyph, out StringBuilder name, out uint size);
```

Comment:
Change StringBuilder to string with UnmanagedType.LPStr

Reply:
I don't think the size is an `out`. It looks to be calculated/assumed somewhere first: https://github.com/harfbuzz/harfbuzz/blob/2.5.2/src/hb-font.h#L507-L510